vulcan/__init__.py

```diff
@@ -1,11 +1,10 @@
 # -*- coding: utf-8 -*-
 
-from __future__ import unicode_literals
+from ._account import Account
+from ._hebe_client import VulcanHebe
+from ._keystore import Keystore
 
-from ._vulcan import Vulcan
-from .hebe import VulcanHebe, Keystore, Account
-
-__version__ = "1.3.1"
+__version__ = "2.0"
 __doc__ = "Unofficial API for UONET+ e-register"
 
-__all__ = ["Vulcan", "VulcanHebe", "Keystore", "Account"]
+__all__ = ["VulcanHebe", "Keystore", "Account"]
```

Comment:
Please change this class name to just `Vulcan`. In v2.0 we want hebe to be the main, and the only one supported API, so you can just remove all the `hebe` words from the whole code.